db/compaction/compaction_outputs.cc

```diff
@@ -587,6 +588,13 @@ Status CompactionOutputs::AddRangeDels(
   for (it->SeekToFirst(); it->Valid(); it->Next()) {
     auto tombstone = it->Tombstone();
     auto kv = tombstone.Serialize();
+    // Filter out by seqno for per-key placement
+    // TODO: should this go lower down?
```

Comment:
I think here is fine since this is not actually "dropping" a tombstone but just to assign to a different level.

Reply:
My concern was overlooking some local state in this function that might be relevant to processing tombstones going to the other output level. I'll re-inspect and remove the comment if it still looks ok.